tests/e2e/specs/frontend/checkout.test.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * External dependencies
+ */
+import {
+	merchant,
+	createSimpleProduct,
+	setCheckbox,
+	settingsPageSaveChanges,
+	verifyCheckboxIsSet,
+} from '@woocommerce/e2e-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getNormalPagePermalink,
+	shopper,
+	visitPostOfType,
+} from '../../../utils';
+
+const block = {
+	name: 'Checkout',
+};
+
+const simpleProductName = 'Woo Single #1';
+const singleProductPrice = '$21.99';
+const twoProductPrice = '$43.98';
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
```

Comment:
is this equivalent to 
```

if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
	// eslint-disable-next-line jest/no-focused-tests
	test.only( `skipping ${ block.name } tests`, () => {} );
}

is this something that can be removed?
```

Reply:
We have the same in [Cart](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/backend/cart.test.js#L26-L28) & [Checkout](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/backend/checkout.test.js#L25-L27) backend tests and [Cart frontend tests](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/frontend/cart.test.js#L25-L27). This is required so in tests that simulate the core package (not the feature plugin) those tests are not run given that Cart and Checkout blocks are not available, so I don't think this can be removed.